examples/amgx/SampleOutput.md

```diff
@@ -0,0 +1,551 @@
+         Grid Complexity: 1.22092
+         Operator Complexity: 1.57763
+         Total Memory Usage: 0.0205989 GB
+         --------------------------------------------------------------
+           iter      Mem Usage (GB)       residual           rate
+         --------------------------------------------------------------
+            Ini            0.882812   1.654940e-02
+              0            0.882812   7.353277e-02         4.4432
+              1              0.8828   1.275661e-02         0.1735
+              2              0.8828   2.186579e-03         0.1714
+              3              0.8828   4.063204e-04         0.1858
+              4              0.8828   7.610406e-05         0.1873
+              5              0.8828   1.397502e-05         0.1836
+              6              0.8828   2.555527e-06         0.1829
+              7              0.8828   4.505549e-07         0.1763
+              8              0.8828   7.845767e-08         0.1741
+              9              0.8828   1.440570e-08         0.1836
+             10              0.8828   2.642394e-09         0.1834
+             11              0.8828   4.788019e-10         0.1812
+             12              0.8828   8.413305e-11         0.1757
+             13              0.8828   1.473831e-11         0.1752
+             14              0.8828   2.652839e-12         0.1800
+             15              0.8828   4.872877e-13         0.1837
+             16              0.8828   9.266083e-14         0.1902
+             17              0.8828   1.770528e-14         0.1911
+         --------------------------------------------------------------
+         Total Iterations: 18
+         Avg Convergence Rate:                   0.2163
+         Final Residual:                                   1.770528e-14
+         Total Reduction in Residual:                         1.069845e-12
+         Maximum Memory Usage:                                            0.883 GB
+         --------------------------------------------------------------
+Total Time: 0.0446627
+    setup: 0.0282668 s
+    solve: 0.0163959 s
+    solve(per iteration): 0.000910882 s
+
+
+Sample 4
+
+lrun -n 4 ./amgx_ex1p --amgx-file amg_pcg.json --amgx-mpi-gpu-exclusive
+Options used:
+   --mesh ../../data/star.mesh
+   --order 1
+   --no-static-condensation
+   --no-partial-assembly
+   --amgx-file amg_pcg.json
+   --amgx-mpi-gpu-exclusive
+   --device cpu
+   --visualization
+   --nd 1
+Device configuration: cpu
+Memory configuration: host-std
+Mesh::GeneratePartitioning(...): edgecut = 169
+Number of finite element unknowns: 82561
+AMGX version 2.1.0.131-opensource
+Built on Sep 25 2020, 10:59:10
+Compiled with CUDA Runtime 10.1, using CUDA driver 10.2
+Cannot read file as JSON object, trying as AMGX config
+Converting config string to current config version
+Parsing configuration string: exception_handling=1 ; 
+Using Normal MPI (Hostbuffer) communicator...
+AMG Grid:
```

Comment:
This is probably more related to AMGx than your PR, but is there a way to not duplicate this output on every processor? It's especially annoying in the iteration monitor output below.

Reply:
I'll look around and see if we can do anything about that.